app/src/test/java/com/example/benchproject/app/movie/details/viewModel/MovieDetailsViewModelTest.kt

```diff
@@ -109,8 +105,6 @@ internal class MovieDetailsViewModelTest {
         coEvery { getMovieDetailsUseCase(movieId) } returns Result.success(movieDetailsModel)
```

Comment:
Names of the tests should be adjusted accordingly :)

Reply:
I changed tests names :D